add luma adjust playback filter

dericed opened the pull request:
resolves https://github.com/bavc/qctools/issues/378

kieranjol commented:
This is brilliant, I wasn't expecting the waveform there as well which makes it even better. I've already been able to use this to confirm that there were details between 235-254 that could have been recovered.
Just one bit of feedback, when clicking on the up/down buttons next to contrast, there should be a finer increment. Right now it's just jumping up by a whole number, when it should be .01 of an increment.
Great stuff!!

kieranjol commented:
Also is it intended that you can't have field and waveform available at the same time? I wouldn't want to use it I don't think, but just thought I'd check.

dericed commented:
Thanks @kieranjol I forgot about slider incrementation there and fixed it locally. I also changed the view when waveform and fieldsplit are both active so now there's a stack of both separate fields and separate waveforms.

kieranjol commented:
Thanks Dave, looking forward to seeing this merged.

dericed commented:
@kieranjol Enjoy!

kieranjol commented:
:))))
